Kitodo/src/main/java/de/sub/goobi/helper/BatchStepHelper.java

```diff
@@ -551,58 +545,51 @@ public String batchDurchBenutzerZurueckgeben() {
                     serviceManager.getMetadataValidationService().setAutoSave(true);
-                    if (!serviceManager.getMetadataValidationService().validate(s.getProcess())) {
+                    if (!serviceManager.getMetadataValidationService().validate(task.getProcess())) {
                         error = true;
                     }
                 }
-                if (s.isTypeImagesWrite()) {
+                if (task.isTypeImagesWrite()) {
                     MetadatenImagesHelper mih = new MetadatenImagesHelper(null, null);
-                    try {
-                        if (!mih.checkIfImagesValid(s.getProcess().getTitle(),
-                                serviceManager.getProcessService().getImagesOrigDirectory(false, s.getProcess()))) {
-                            error = true;
-                        }
-                    } catch (Exception e) {
-                        Helper.setFehlerMeldung("Error on image validation: ", e);
+                    if (!mih.checkIfImagesValid(task.getProcess().getTitle(),
+                            serviceManager.getProcessService().getImagesOrigDirectory(false, task.getProcess()))) {
+                        error = true;
+                        Helper.setFehlerMeldung("Error on image validation!");
                     }
                 }
 
-                loadProcessProperties(s);
+                loadProcessProperties(task);
 
                 for (Property prop : this.properties) {
-                    if ((prop.getValue() == null || prop.getValue().equals(""))) {
-                        List<String> parameter = new ArrayList<>();
-                        parameter.add(prop.getTitle());
-                        parameter.add(s.getProcess().getTitle());
-                        Helper.setFehlerMeldung(Helper.getTranslation("BatchPropertyEmpty", parameter));
+                    if (isPropertyInvalid(prop, task)) {
                         error = true;
                     }
                 }
             }
             if (!error) {
-                this.myDav.uploadFromHome(s.getProcess());
-                TaskDAO taskDAO = new TaskDAO();
-                Task task = taskDAO.getById(s.getId());
+                this.myDav.uploadFromHome(task.getProcess());
                 task.setEditTypeEnum(TaskEditType.MANUAL_MULTI);
-                serviceManager.getWorkflowControllerService().close(s);
+                serviceManager.getWorkflowControllerService().close(task);
             }
         }
         AktuelleSchritteForm asf = (AktuelleSchritteForm) Helper.getManagedBeanValue("#{AktuelleSchritteForm}");
         return asf.filterAll();
     }
 
-    /**
-     * Get script' names.
-     *
-     * @return list of names
-     */
-    public String getScriptName() {
-        return getCurrentStep().getScriptName();
+    private boolean isPropertyInvalid(Property property, Task task) {
+        if ((property.getValue() == null || property.getValue().equals(""))) {
```

Comment:
Braces are correct set as you double encode the expression. It is only moved code but can be fixed to
```java
if (property.getValue() == null || property.getValue().equals("")) {
....
}
```